source/retryable-reads/retryable-reads.rst

```diff
@@ -79,7 +79,7 @@ SocketException                 9001
 
   .. _PoolClearedError: ../connection-monitoring-and-pooling/connection-monitoring-and-pooling.rst#connection-pool-errors
 
-MongoClient Configuration 
+MongoClient Configuration
```

Comment:
If you're trimming trailing whitespace I think you also need to adjust the heading line below to avoid RST warnings. Not just here but in the trimmed headings later in this file.

Reply:
What warnings are you referring too, is there some local tooling that checks the rst format?

Reply:
I'm probably thinking of tooling used to build the MongoDB manual. I don't believe anything in the specs repo checks for this.

See: https://docutils.sourceforge.io/docs/ref/rst/restructuredtext.html#sections:

> When an overline is used, the length and character used must match the underline.

Reply:
Ah! the actual underline length should match, I didn't connect the two for some reason, fixed!